packages/web-components/@sfx/autocomplete/test/autocomplete.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, spy, stub } from './utils';
+import { Base } from '@sfx/base';
+import { TemplateResult } from 'lit-element';
+import Autocomplete from '../src/autocomplete';
+
+describe('Autcomplete Component', () => {
+  let autocomplete;
+  beforeEach(() => {
+    autocomplete = new Autocomplete();
+  });
+
+  describe('Constructor', () => {
+    it('should extend the Base class', () => {
+      expect(autocomplete).to.be.an.instanceof(Base);
+    });
+  });
+
+  describe('connectedCallback', () => {
+    it('should add an eventListener to the window', () => {
+      spy(window, 'addEventListener');
```

Comment:
In general, you should control as much as you can in your unit test. In this case, I would save this spy to a variable and use that variable in the expectation instead of going through `window`, because (although unlikely) `connectedCallback()` _could_ modify `addEventListener`.